generic_config_updater/gu_common.py

```diff
@@ -162,6 +166,32 @@ def validate_field_operation(self, old_config, target_config):
                 if any(op['op'] == operation and field == op['path'] for op in patch):
                     raise IllegalPatchOperationError("Given patch operation is invalid. Operation: {} is illegal on field: {}".format(operation, field))
 
+        def _invoke_validating_function(cmd):
+            # cmd is in the format as <package/module name>.<method name>
+            method_name = cmd.split(".")[-1]
+            module_name = ".".join(cmd.split(".")[0:-1])
+            module = importlib.import_module(module_name, package=None)
+            method_to_call = getattr(module, method_name)
+            return method_to_call()
```

Comment:
>method_to_call [](http://example.com/codeflow?start=19&length=14)

There is potential risk to run ambiguous code. We can proactively check the module, and method_names. Maybe we can have allowlist or similar?

Reply:
Yes, I added code to ensure the module_name is `generic_config_updater.field_operation_validators` and the `method_name` contains the string `validator`